keras/src/layers/attention/additive_attention.py

```diff
@@ -33,7 +33,7 @@ class AdditiveAttention(Attention):
             - `query`: Query tensor of shape `(batch_size, Tq, dim)`.
             - `value`: Value tensor of shape `(batch_size, Tv, dim)`.
             - `key`: Optional key tensor of shape `(batch_size, Tv, dim)`. If
-                not given, will use `value` for both `key` and `value`, which is
+                not given, will use the`value` for both `key` and `value`, which is
```

Comment:
The line you modified is now 83 characters long, which is why the code format check didn't pass. The maximum allowed length is 80.
You can rewrite it to:
```suggestion
                not given, will use the`value` for both `key` and `value`, 
                which is the most common case.
```

<img width="458" alt="image" src="https://github.com/user-attachments/assets/60d04ff9-1e7d-4a1c-8362-35ef25308b12" />